solutions/serializers.py

```diff
@@ -0,0 +1,31 @@
+from rest_framework import serializers
+from missions.serializers import MissionSerializer
+from .models import SolutionHistory, Message
+
+class SolutionHistorySerializer(serializers.ModelSerializer):
+    class Meta:
+        model = SolutionHistory
+        fields = "__all__"
+
+class SolutionHistoryStatusUpdateSerializer(serializers.Serializer):
+    is_solved = serializers.BooleanField()
+    
+
+class SolutionHistoryListSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = SolutionHistory
+        fields = ['id', 'created_at', 'is_solved', 'updated_at']
```

Comment:
❗ [Code Convention](https://www.notion.so/Code-Convention-276cc780cd7d806ea542fe3b9748257d?source=copy_link#276cc780cd7d80c1b071f39380a79c7b)을 지켜 주세요.
```suggestion
        fields = ('id', 'created_at', 'is_solved', 'updated_at',)
```